diff --git a/dist/jsx/hooks/index.js b/dist/jsx/hooks/index.js
index ec842c73a8426a94201a777ec1e909695ec44e68..d1eaae888dc795faad80a21ce52f2514a1d72167 100644
--- a/dist/jsx/hooks/index.js
+++ b/dist/jsx/hooks/index.js
@@ -300,7 +300,7 @@ var useSyncExternalStore = (subscribe, getSnapshot, getServerSnapshot) => {
     return subscribe(() => {
       setState(getSnapshot());
     });
-  }, []);
+  }, [subscribe]);
   return state;
 };
 export {
